azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md

```diff
@@ -0,0 +1,45 @@
+---
+external help file: Microsoft.Open.MS.GraphV10.PowerShell.dll-Help.xml
+Module Name: AzureAD
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureADMSAuthorizationPolicy
+
+## SYNOPSIS
+Gets an authorization policy.
+
+## SYNTAX
+
+```
+Get-AzureADMSAuthorizationPolicy [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-AzureADMSAuthorizationPolicy cmdlet gets an Azure Active Directory authorization policy.
+
+## EXAMPLES
+
+### Example 1: Get an authorization policy by ID
+```
+PS C:\>Get-AzureADMSAuthorizationPolicy
+```
+
+This commands gets the Azure AD authorization policy.
+
+## PARAMETERS
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
```

Comment:
Sections with missing content. Is this typical? Can we remove the sections with missing info or no?